### PR TITLE
Schedules 2nd iteration updates

### PR DESCRIPTION
**Pull request by maskin25**
# What this PR does

Features and bugs related to [[Q1 2023] Iteration with Schedules](https://github.com/grafana/oncall-private/issues/1660) milestone

## Checklist

- [x] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

**Comment by Konstantinov-Innokentii** [comment]
Please, do not merge part with rotation title untill https://github.com/grafana/oncall/pull/1833 will be merged.

File: grafana-plugin/src/containers/Rotation/Rotation.tsx [hunk 1]
```diff
@@ -23,37 +23,52 @@ interface RotationProps {
...
 
-    onClick(startMoment.add(dayOffset, 'day'));
+    const shiftStart = startMoment.add(dayOffset, 'day');
+    const shiftEnd = shiftStart.add(1, 'day');
+
+    onClick(shiftStart, shiftEnd);
+  };
+
+  const getAddOverrideClickHandler = (scheduleEvent) => {
+    return (event) => {
```
Comment (teodosii) [inline grafana-plugin/src/containers/Rotation/Rotation.tsx:67]: Here too

File: grafana-plugin/src/containers/RotationForm/RotationForm.tsx [hunk 2]
```diff
@@ -66,84 +86,78 @@ const RotationForm: FC<RotationFormProps> = observer((props) => {
...
+
+  const [rotationStart, setRotationStart] = useState<dayjs.Dayjs>(shiftStart);
+  const [endLess, setEndless] = useState<boolean>(true);
+  const [rotationEnd, setRotationEnd] = useState<dayjs.Dayjs>(shiftStart.add(1, 'month'));
+
   const [repeatEveryValue, setRepeatEveryValue] = useState<number>(1);
   const [repeatEveryPeriod, setRepeatEveryPeriod] = useState<number>(0);
+
+  const [showActiveOnSelectedDays, setShowActiveOnSelectedDays] = useState<boolean>(false);
+  const [showActiveOnSelectedPartOfDay, setShowActiveOnSelectedPartOfDay] = useState<boolean>(false);
```
[the remark this answers is not in the document]
Comment (maskin25) [reply grafana-plugin/src/containers/RotationForm/RotationForm.tsx:126]: good idea, not sure if I have time atm, I propose to postpone this refactoring a bit

File: grafana-plugin/src/containers/Rotation/Rotation.tsx [hunk 3]
```diff
@@ -23,37 +23,52 @@ interface RotationProps {
...
+    onClick,
+    handleAddOverride,
+    simplified,
+    filters,
   } = props;
 
   const [animate, _setAnimate] = useState<boolean>(true);
 
-  const handleClick = (event) => {
+  const handleRotationClick = (event) => {
```
[the remark this answers is not in the document]
Comment (maskin25) [reply grafana-plugin/src/containers/Rotation/Rotation.tsx:54]: fixed

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- iCal export final schedule oncall [#1501](https://github.com/grafana/oncall/issues/1501)
+- Schedules: Allow web-edited overrides for Terraform-managed rotations [828](https://github.com/grafana/oncall/issues/828)
```
Comment (matiasb) [inline CHANGELOG.md:13]: FYI, these 2 above were already released and available (so I would remove them from here)

**Review by matiasb — APPROVED** [review]

**Pull request merged**